.github/workflows/grid_client_nightly.yml

```diff
@@ -91,6 +91,23 @@ jobs:
         run: |
           yarn run ts-node --project packages/grid_client/tsconfig-node.json packages/grid_client/scripts/delete_all_contracts.ts
 
+      - name: Run check up - List all contracts
+        id: command_step
+        run: |
+          sleep 15
+          yarn run ts-node --project packages/grid_client/tsconfig-node.json packages/grid_client/scripts/list_all_contracts.ts > output.txt
+
+      - name: Check if contracts are Empty
+        run: |
+          # Read the Contracts line from the file
+          contracts_json=$(awk 'NR==5' output.txt)
+
+          # Check if all three [Name - Node - Rent] contracts are empty
+          if [[ $contracts_json != *'nodeContracts: []'* ]]; then
+            echo "Deletion failed; The Node Contract is not empty."
+            exit 1
+          fi
```

Comment:
When the this check fails it just prints a message without mentioning the failed contracts.

![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/101194226/80c7be4a-275d-41bf-a0d9-7cf7fdbeaec9)

Imo it should mention the contracts that weren't deleted if possible and in case of failure it should re-run the delete all scripts again to ensure that the clean up was sucessfull.

Reply:
There was a test for the delete all contract; if it fails, the workflow fails, and then I added the delete checker to make sure that even if the delete all task was successful, all the contracts should be checked to make sure the test is not just passing, and if not, print the failing to delete contract.

The workflow's main focus isn't to retry deleting that many but rather to notify us if those jobs are failing and the reason (especially since we don't want the delete script to just give wrong information about a successful deletion and the contracts are deleted as said).